mLRS/Common/hal/esp/tx-hal-jumper-int-2400-esp32.h

```diff
@@ -0,0 +1,175 @@
+#define UART_USE_RX_IO            IO_P3
+#define UART_TXBUFSIZE            TX_SERIAL_TXBUFSIZE  // 512
+#define UART_RXBUFSIZE            TX_SERIAL_RXBUFSIZE  // 2048
+
+
+//-- SX1: SX12xx & SPI
+
+#define SPI_CS_IO                 IO_P5
+#define SPI_MISO                  IO_P19
+#define SPI_MOSI                  IO_P23
+#define SPI_SCK                   IO_P18
+#define SPI_FREQUENCY             18000000L
+#define SX_RESET                  IO_P14
+#define SX_BUSY                   IO_P21
+#define SX_DIO1                   IO_P4
+#define SX_TX_EN                  IO_P26
+#define SX_RX_EN                  IO_P27
+#define PWR_EN                    IO_P25
+
+IRQHANDLER(void SX_DIO_EXTI_IRQHandler(void);)
+
+void sx_init_gpio(void)
+{
+    gpio_init(SX_DIO1, IO_MODE_INPUT_ANALOG);
+    gpio_init(SX_BUSY, IO_MODE_INPUT_PU);
+    gpio_init(SX_TX_EN, IO_MODE_OUTPUT_PP_LOW);
+    gpio_init(SX_RX_EN, IO_MODE_OUTPUT_PP_LOW);
+    gpio_init(SX_RESET, IO_MODE_OUTPUT_PP_HIGH);
+    gpio_init(PWR_EN, IO_MODE_OUTPUT_PP_HIGH);  // front-end enable
+}
+
+IRAM_ATTR bool sx_busy_read(void)
+{
+    return (gpio_read_activehigh(SX_BUSY)) ? true : false;
+}
+
+IRAM_ATTR void sx_amp_transmit(void)
+{
+    gpio_low(SX_RX_EN);
+    gpio_high(SX_TX_EN);
+}
+
+IRAM_ATTR void sx_amp_receive(void)
+{
+    gpio_low(SX_TX_EN);
+    gpio_high(SX_RX_EN);
+}
+
+void sx_dio_enable_exti_isr(void)
+{
+    attachInterrupt(SX_DIO1, SX_DIO_EXTI_IRQHandler, RISING);
+}
+
+void sx_dio_init_exti_isroff(void)
+{
+    detachInterrupt(SX_DIO1);
+}
+
+void sx_dio_exti_isr_clearflag(void) {}
+
+
+//-- Button
+
+#define BUTTON                    -1
+
+void button_init(void) {}
+
+IRAM_ATTR bool button_pressed(void) { return false; }
+
+
+//-- LEDs
+
+#define LED_RED                   -1
+
+void leds_init(void) {}
+
+IRAM_ATTR void led_red_off(void) {}
+IRAM_ATTR void led_red_on(void) {}
+IRAM_ATTR void led_red_toggle(void) {}
+
+
+//-- Cooling Fan
+
+#define FAN_IO                    IO_P33
+
+void fan_init(void)
+{
+    gpio_init(FAN_IO, IO_MODE_OUTPUT_PP_LOW);
+}
+
+IRAM_ATTR void fan_set_power(int8_t power_dbm)
+{
+    if (power_dbm >= POWER_23_DBM) {
+        gpio_high(FAN_IO);
+    } else {
+        gpio_low(FAN_IO);
+    }
+}
+
+
+//-- ESP32 Wifi Bridge
+
+#ifdef DEVICE_HAS_ESP_WIFI_BRIDGE_ON_SERIAL
+
+#define ESP_RESET                 IO_P15 // backpack_en
+#define ESP_GPIO0                 IO_P2  // backpack_boot inverted?
+
+void esp_init(void)
+{
+    gpio_init(ESP_GPIO0, IO_MODE_OUTPUT_PP_LOW); // high -> esp will start in bootloader mode
+    gpio_init(ESP_RESET, IO_MODE_OUTPUT_PP_LOW); // low -> esp is in reset
+}
+
+IRAM_ATTR void esp_reset_high(void) { gpio_high(ESP_RESET); }
+IRAM_ATTR void esp_reset_low(void) { gpio_low(ESP_RESET); }
+
+IRAM_ATTR void esp_gpio0_high(void) { gpio_low(ESP_GPIO0); }
+IRAM_ATTR void esp_gpio0_low(void) { gpio_high(ESP_GPIO0); }
+
+#endif
+
+
+//-- POWER
+#define POWER_GAIN_DBM            28 // gain of a PA stage if present
```

Comment:
is this a known PA, which could be used from https://github.com/olliw42/mLRS/blob/main/mLRS/Common/hal/hal-power-pa.h ?